pykylin/cursor.py

```diff
@@ -57,11 +100,13 @@ def _type_mapped(self, result):
             if tpe == 'DATE':
                 val = parser.parse(val)
             elif tpe == 'BIGINT' or tpe == 'INT' or tpe == 'TINYINT':
-                val = int(val)
+                val = int(val) if val else 0
```

Comment:
That seems OK to me, but we have to add some test cases to testify the None value in Caravel.